app/views/genres/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% @genres.each do |genre| %>
+  <%= link_to genre.name, genre_path(genre) %>
+<% end %>
+
+<% if current_user && current_user.admin? %>
```

Comment:
Hint: You have `current_admin?` defined in ApplicationController, but it's not exposed as a method that can be used outside of other controllers.